src/LaravelEditorJsServiceProvider.php

```diff
@@ -38,5 +39,15 @@ public function boot()
                 __DIR__ . '/../resources/views' => $this->app->resourcePath('views/vendor/laravel_editorjs'),
             ], 'laravel_editorjs-views');
         }
+
+        /**
+         * Blade directive to render editor.js blocks.
+         *
+         * @param mixed $blocks The blocks of content to be rendered.
+         * @return string Rendered HTML content.
+         */
+        Blade::directive('render', function ($blocks) {
+            return "<?php echo app('laravel-editorjs')->render($blocks); ?>";
```

Comment:
@lmottasin What about the exception?

`render()` method throws exception when it fails to render.

Reply:
[I thought you handled this ](https://github.com/alaminfirdows/laravel-editorjs/issues/21)

Reply:
What I found is that handling the exception is not a valid solution. The developer must validate the JSON before saving it to the database, so the current approach is fine.